### PR TITLE
[Language] Remove duplication for PreBuiltResult and InputError

### PR DESCRIPTION
**Pull request by minhanh-phan**
# Data Plane API - Pull Request
`PreBuiltResult` and `RequestStatistics` break autorest schema check because they are duplicated in both `common.json` and `analyzetext.json`. Edit all the `analyzetext.json` reference to point to `common.json`. Unify `InputError` with `DocumentError`.

## API Info: The Basics
Most of the information about your service should be captured in the issue that serves as your [*API Spec engagement record*](https://aka.ms/azsdk/onboarding/restapischedule).

* Link to API Spec engagement record issue:

Is this review for (select one):

- [ ] a private preview
- [ ] a public preview
- [ ] GA release

### Change Scope
<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous API Spec document (if applicable), and the root paths that have been updated. </sup>
* Design Document:
* Previous API Spec Doc:
* Updated paths:

## ❔Got questions? Need additional info?? We are here to help!

<details>
  <summary> Contact us!</summary>

The [Azure API Review Board](https://aka.ms/azsdk/onboarding/restapischedule) is dedicated to helping you create amazing APIs. You can read about our mission and learn more about our process on our [wiki](https://aka.ms/azsdk/onboarding/restapischedule).
* 💬 [Teams Channel](https://teams.microsoft.com/l/channel/19%3a3ebb18fded0e47938f998e196a52952f%40thread.tacv2/General?groupId=1a10b50c-e870-4fe0-8483-bf5542a8d2d8&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
* 💌 [email](mailto://azureapirbcore@microsoft.com)

</details>

<details>
  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>

### Tooling
 * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
 * [Spectral Linting](https://aka.ms/style)
 * [Open API Hub](https://aka.ms/openapiportal)

### Guidelines & Specifications
 * [Azure REST API Guidelines](https://aka.ms/guidelines)
 * [OpenAPI Style Guidelines](https://aka.ms/style)
 * [Azure Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy)

### Helpful Links
 * [Azure DevTools Wiki](https://aka.ms/azapi)

</details>

### CODE REVIEW

**Review by deyaaeldeen — APPROVED** [review]

**Review by mssfang — APPROVED** [review]

**Review by kristapratico — COMMENTED** [review]
RequestStatistics is defined in analyzetext.json. ~Do we need to remove that definition and ref from common.json?~ Do we need to remove it from analyzetext.json and then use the ref from common.json?

**Comment by joseharriaga** [comment]
> RequestStatistics is defined in analyzetext.json. Do we need to remove that definition and ref from common.json?

Would it be better to do it the other way around: Remove it from analyzetext.json and keep it in common.json?

**Comment by kristapratico** [comment]
> > RequestStatistics is defined in analyzetext.json. Do we need to remove that definition and ref from common.json?
> 
> Would it be better to do it the other way around: Remove it from analyzetext.json and keep it in common.json?

Sorry bad wording, this is what I'm suggesting 😆

**Comment by minhanh-phan** [comment]
@kristapratico Yes, I miss this. Pushed the change. Thank you for noticing that.

**Review by joseharriaga — APPROVED** [review]

**Comment by kristapratico** [comment]
> @kristapratico Yes, I miss this. Pushed the change. Thank you for noticing that.

Generates successfully for me, thank you!

**Review by kristapratico — APPROVED** [review]

**Review by heaths — APPROVED** [review]
I believe this is okay, but checking with @tg-msft on one thing - the effective addition of `additionalProperties: true`.

File: dev/cognitiveservices/data-plane/Language/analyzetext.json [hunk 1]
```diff
@@ -545,29 +545,6 @@
         }
       ]
     },
-    "PreBuiltResult": {
-      "type": "object",
```
Comment (heaths) [inline dev/cognitiveservices/data-plane/Language/analyzetext.json:549]: This is also missing in common.json. I think we should add `type: object` there as well. In practice, it shouldn't make a difference. I think all our modern code generators don't really care, but we should compare generated outputs for any differences.

**Review by heaths — APPROVED** [review]

**Comment by heaths** [comment]
The breaking changes are expected and have not previous GA'd. This is effectively a correction.

**Pull request merged**